airbyte-integrations/connectors/destination-postgres/src/integrationTestLegacy/kotlin/io/airbyte/integrations/destination/postgres/typing_deduping/PostgresTypingDedupingTest.kt

```diff
@@ -45,6 +47,19 @@ open class PostgresTypingDedupingTest : AbstractPostgresTypingDedupingTest() {
     override val imageName: String
         get() = "airbyte/destination-postgres:dev"
 
+    override fun disableRawTableComparison(): Boolean = true
+
+    // older versions of the connector used to error out in this scenario. This is not
+    // true for newer versions of the connector
+    @Disabled @Test override fun interruptedTruncateWithPriorData() {}
```

Comment:
truth be told, I don't 100% understand what this test is doing, I just know it used to fail and doesn't fail anymore. I'm assuming this is the behavior we want.

Reply:
:+1: this is covered by new CDK tests

(specifically: the old CDK threw an error if we reached the end of stdin without seeing a "stream status" message; the new CDK only throws an error if we get an explicit `INCOMPLETE` stream status. In practice, platform _always_ sends an explicit complete/incomplete status, so there's no behavior change at runtime)